### ISSUE TITLE
New problem with Script 3 state 16

### ISSUE THREAD

**Issue by Harpig** [KeeyanGhoreshi/PokemonPlatinumSingleSequence#27]
When we go stuffing Eevee with yucky medicine, a wild Ralts fled before we could run away ourselves, causing us to leave the grass.
[-8__16-AfterSecondTrainer_1338697037.zip](https://github.com/user-attachments/files/17823066/-8__16-AfterSecondTrainer_1338697037.zip)

**Comment by KeeyanGhoreshi** [comment]
interesting, ralts teleporting should be accounted for.

**Comment by KeeyanGhoreshi** [comment]
oh right when aligning after a potential teleport, if you're facing left already the left input causes you to enter the bump state which is somewhat volatile and causes the next movement input to actually move you a tile instead of turning.  The fix (as usual) is adding a couple frames of wait time between the left and down inputs to let things settle before doing another directional input.

**Comment by Harpig** [comment]
Is it cool for me to keep submitting issues? Feels weird being the only one. I don't want to bother you if you want to focus on other things.

**Comment by KeeyanGhoreshi** [comment]
yes, totally fine and I actually encourage it.  I want the sequence to be as robust as possible.
